File: src/main/java/com/twocogs/homeassistant/pcmonitor/ServletInitializer.java
package com.twocogs.homeassistant.pcmonitor;

import com.twocogs.homeassistant.pcmonitor.PcmonitorApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.support.SpringBootServletInitializer;

public class ServletInitializer extends SpringBootServletInitializer {

	@Override
	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
		return application.sources(PcmonitorApplication.class);
	}

}

File: src/main/kotlin/com/twocogs/homeassistant/pcmonitor/ServletInitializer.kt
package com.twocogs.homeassistant.pcmonitor

import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.web.support.SpringBootServletInitializer

class ServletInitializer : SpringBootServletInitializer() {
    override fun configure(application: SpringApplicationBuilder): SpringApplicationBuilder {
        return application.sources(PcMonitorApplication::class.java)
    }
}